pylintrc

```diff
@@ -0,0 +1,29 @@
+[MESSAGES CONTROL]
+disable = 
+	locally-disabled,
+	locally-enabled,
+	no-init,
+	no-self-use,
+	too-few-public-methods,
+	too-many-ancestors,
+	too-many-arguments,
+	too-many-instance-attributes,
+	too-many-public-methods,
+
+[REPORTS]
+output-format = text
+
+[BASIC]
+good-names = f,i,j,k,db,ex,Run,_,__,js
+no-docstring-rgx = __.*__$|test_.+|setUp$|setUpClass$|tearDown$|tearDownClass$|Meta$
+
+[FORMAT]
+ignore-long-lines = ^.*https?://.*$
```

Comment:
In a follow-up PR, this is converted to handle _any_ URL containing double-slashes,
for cases where the protocol is intentionally omitted.